packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -71,8 +69,7 @@ class _DriverBinding extends BindingBase with ServicesBinding, SchedulerBinding,
 /// with an `isError` boolean.
 void enableFlutterDriverExtension({ DataHandler handler, bool silenceErrors = false }) {
   assert(WidgetsBinding.instance == null);
-  _DriverBinding(handler, silenceErrors);
-  assert(WidgetsBinding.instance is _DriverBinding);
+  ExtensionBinding.registerExtensionRegistrar(_DriverExtensionRegistrar(handler, silenceErrors));
 }
 
 /// Signature for functions that handle a command and return a result.
```

Comment:
this needs tests

Reply:
Can you give me some tip that how should i design this test case?
I tried some, but feel hard to create a test case without changing the original api.